docs/housekeeping-api-requests.html

```diff
@@ -50,6 +50,20 @@ <h3>API Requests</h3>
 	</div>
 </div>
 
+<div class="chart-placeholder">
+	<h3>Top API Requests by Users</h3>
```

Comment:
I’d drop the “Top” in the title, because it’s common for our charts only to show the top entries (for instance, in the [Git traffic table](https://autodesk.github.io/hubble/housekeeping-git-traffic)).